Assets/Scripts/Animals/Scared.cs

```diff
@@ -0,0 +1,58 @@
+// This script is responsible for activating the warning and the movement when a scared animal detects the player
+// Created by: Palin Wiseman
+// Created on: 23/08/2023
+// Last modified by: Palin Wiseman
+// Last modified on: 23/08/2023
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Scared : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject warningSymbol;
+
+    private AnimalController animalController;
+    private GameObject player;
+
+    private bool alerted;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        warningSymbol.SetActive(false);
+        alerted = false;
+        animalController = GetComponentInParent<AnimalController>();
+        player = GameObject.FindWithTag("Player");
+    }
+
+    void Update()
+    {
+        if (alerted) //If the animal is alerted to the player then turn on the warning symbol and activate the movement
+        {
+            animalController.Scared(player.transform.position);
+            warningSymbol.SetActive(true);
```

Comment:
Seems to flash very briefly until the animal turns away.

Reply:
Addressed. Now lasts for 10 seconds.